setup/security/authorization/pipeline-permissions/index.md

```diff
@@ -56,4 +56,35 @@ the previously selected service account in order to maintain backwards
 compatibility. Newly added triggers will use the permissions specified. If you
 wish to use pipeline permissions for the older triggers and already have
 permissions specified, edit your pipeline JSON and remove the `RunAsUser` field
-from your trigger.
+from your trigger, or enable the automatic migration (see next section).
+
+### Automatic migration
+
+Front50 can automatically migrate all pipelines from using Fiat service accounts to
```

Comment:
@jervi 
Maybe link "Fiat service accounts" to https://www.spinnaker.io/setup/security/authorization/service-accounts/ ?